kubernetes/browser.yaml

```diff
@@ -4,55 +4,62 @@ metadata:
   name: browser
   namespace: buildbarn
 spec:
-  replicas: 3
+  replicas: 1
   selector:
     matchLabels:
       app: browser
   template:
     metadata:
       labels:
         app: browser
+        cluster: buildbarn
     spec:
+      nodeSelector:
+        agentpool: workerpool01
       containers:
-      - image: buildbarn/bb-browser:20200816T120922Z-6dc5c54
-        args:
-        - /config/browser.jsonnet
-        name: browser
-        ports:
-        - containerPort: 80
-          protocol: TCP
-        volumeMounts:
-        - mountPath: /config/
-          name: configs
-          readOnly: true
+        - image: buildbarn/bb-browser:20210811T090408Z-9f5dd80
```

Comment:
Could you change the indentation back to the original? I think this is also the way kubectl's YAML parser/printer formats this stuff.